### PR TITLE
Api rooms doc

### LINKED ISSUE #37: API-Docs-User/Hotel/Room/Auth
- [x] Update error messages on hotel endpoints to match the errorMessage object
- [x] Add docs for any undocumented endpoints
- [x] Update/check docs across all endpoints for consistency

### PR DESCRIPTION
**Pull request by sampler36**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please add a relevant label.

## Change status

- [x] Complete, tested, ready to review and merge

- [x] Complete, but not tested (may need new tests)

- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe steps taken to ensure this feature is functional and does not break other features:

# Checklist:

- [x] My code follows the style guidelines of this project

- [x] I have performed a self-review of my own code

- [x] I have commented my code, particularly in hard-to-understand areas

- [x] I have made corresponding changes to the documentation, if relevant

- [x] I have run the app using my feature and ensured that no functionality is broken

- [x] My changes generate no new warnings

- [x] There are no merge conflicts

Resolve #43 
Resolved #37

### CODE REVIEW

File: routes/endpoint-docs/roomRoutesDocs.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+ *      "name": "1",
+ *      "id": "5cc95fa83de71d13395320fa",
+ *    }
+ *
+ *  @apiSuccess {String} _id hotel Id
+ *  @apiSuccess {String} name of the new room
+ *  
+ *  @apiSuccessExample Success-Response: Room created
+ *    HTTP/1.1 201 OK
+ *  ...... This will appear on the bottom of the list when you do               a get request after you put
```
Comment (mark-marshall) [inline routes/endpoint-docs/roomRoutesDocs.js]: Slightly weird spaces on line 138

File: routes/endpoint-docs/roomRoutesDocs.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+ *   {
+ *   "_id": "5cc95fa83de71d13395320fd",
+ *   "name": "4",
+ *    }
+ * ]  ..............till the last room
+ * 
+ *  @apiErrorExample Error-Response: Wrong Unique Id
+ *    HTTP/1.1 400 BAD REQUEST
+ *    {
+ *       "message": "An invalid ObjectId was passed."
```
Comment (mark-marshall) [inline routes/endpoint-docs/roomRoutesDocs.js:42]: For each of the endpoints, can you run through and add all possible error messages? I think we have 1 for each but in reality, there should be 3-5 ish per endpoint.

File: routes/endpoint-docs/userRoutesDocs.js [hunk 3]
```diff
@@ -68,6 +68,11 @@
  * @apiGroup Users
  *
  * @apiParam {Number} _id Users unique ID.
+ * @apiParamsExample  {} Request-Example:
+ *    {
+ *   
+ *    }
```
Comment (mark-marshall) [inline routes/endpoint-docs/userRoutesDocs.js:74]: Ok to leave this blank for now while User endpoints are being rebuilt, just remember its there!

**Review by mark-marshall — CHANGES_REQUESTED** [review]
Just a couple of comments, but otherwise great job man 🎯

File: routes/endpoint-docs/userRoutesDocs.js (hunk 3 above)
Comment (sampler36) [reply routes/endpoint-docs/userRoutesDocs.js:74]: Sorry I left it the time you said Pavol was working on it

**Review by sampler36 — COMMENTED** [review]
done some requested changes

**Review by mark-marshall — APPROVED** [review]

**Pull request merged**